src/server/system_services/bucket_server.js

```diff
@@ -412,12 +411,19 @@ async function get_bucket_logging(req) {
 
     dbg.log0('get_bucket_logging:', req.rpc_params);
     const bucket = find_bucket(req);
+    let logging;
 
-    const logging = {
-        "name": bucket.name,
-        "log_bucket": bucket.logging.log_bucket,
-        "log_prefix": bucket.logging.log_prefix
-    };
+    if (bucket.logging && bucket.logging.log_bucket) {
+        logging = {
+            "log_bucket": bucket.logging.log_bucket,
+            "log_prefix": bucket.logging.log_prefix
+        };
+    } else {
+        logging = {
+            "log_bucket": "Not Configured",
+            "log_prefix": "Not Configured"
+        };
+    }
```

Comment:
Maybe use conditional formatting? 
```suggestion
    const logging = {
        "log_bucket": bucket.logging?.log_bucket || "Not Configured",
        "log_prefix": bucket.logging?.log_prefix || "Not Configured"
    };
```

Reply:
done